server/app.js

```diff
@@ -193,17 +192,22 @@ module.exports = function createApp({ signInService, formService }) {
     res.redirect(authLogoutUrl)
   })
   app.use(authenticationMiddleware())
-  app.get('/', (req, res) => res.render('formPages/crn_search'))
-  app.get('/offender_summary/:crn(x\\d{6})', (req, res) => {
+  app.get('/', (req, res) => res.render('formPages/crnSearch'))
+  app.get('/offenderSummary/crn/:crn(x\\d{6})', (req, res) => {
```

Comment:
I don't know the context and nothing to do with me, but for our pages urls should normally be separated by hyphens and all lower case - https://www.gov.uk/guidance/content-design/url-standards-for-gov-uk

Reply:
We had that debate yesterday, didn't spot the official guidance, just looked at the other projects. I'll update to follow the official guidance 👍